chromium_src/ios/web/navigation/crw_wk_navigation_handler.mm

```diff
@@ -46,6 +46,19 @@ - (void)webView:(WKWebView*)webView
           preferences.allowsContentJavaScript = false;
         }
 
+        if (@available(iOS 18, *)) {
+        } else {
```

Comment:
this is a bit weird, why the empty block?

Reply:
There's no "unavailable" check in Obj-C like we have in Swift and we can't apply a negation on it (`if (!@available(iOS 18, *)) { ... }` is not valid syntax)